integrationExamples/gpt/hello_world.html

```diff
@@ -20,15 +20,15 @@
             code: 'div-gpt-ad-1460505748561-0',
             mediaTypes: {
                 banner: {
-                    sizes: [[300, 250]],
+                    sizes: [[320, 250]],
                 }
             },
 
             // Replace this object to test a new Adapter!
             bids: [{
-                bidder: 'appnexus',
+                bidder: 'brianx',
                 params: {
-                    placementId: 13144370
+                    pubId: 'F7B53DBC-85C1-4685-9A06-9CF4B6261FA3'
                 }
```

Comment:
please remove changes to this file in pr

Reply:
Already restored